apps/design/frontend/src/features_context.tsx

```diff
@@ -33,6 +33,17 @@ enum UserFeature {
    * Allow the user to delete an election.
    */
   DELETE_ELECTION = 'DELETE_ELECTION',
+  /**
+   * Allow the user to change the splits for a precinct split. The goal of
+   * this feature flag is to prevent users from accidentally messing up preset
+   * precinct splits.
+   */
+  CREATE_DELETE_PRECINCT_SPLITS = 'CREATE_DELETE_PRECINCT_SPLITS',
+  /**
+   * Allow the user to create and delete districts. The goal of this feature
+   * flag is to prevent users from accidentally messing up preset districts.
+   */
+  CREATE_DELETE_DISTRICTS = 'CREATE_DELETE_DISTRICTS',
```

Comment:
Am recalling that we should also probably disallow creating and deleting top-level precincts, not just splits

Reply:
ee48d5d6e